.travis.yml

```diff
@@ -1,44 +1,24 @@
-language: generic
+language: node_js
 sudo: required 
 services: 
   - docker
 branches:
   only:
-    - master
+    - main
 cache:
   directories:
-    - $HOME/docker
-env:
-  global:
-    - DOCKER_IMAGE='cerner/duplicate-package-checker-webpack-plugin'
-    - TAG="${TRAVIS_PULL_REQUEST_SHA:=latest}"
+    - travis-build
 before_install:
-  - sudo chown -R travis:travis $HOME/docker
-  # Load cached docker images
-  - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
-before_cache:    
-  - sudo chown -R travis:travis $HOME/docker
-  - if [ $TRAVIS_TEST_RESULT == 1 ]; then rm -f $HOME/docker/*.tar.gz; fi
+  # Clean install on any new build
+  - npm run clean
 jobs:
   include:
-    - stage: build
-      before_install: true
-      script:
-        - sudo chown -R travis:travis $HOME/docker
-        - 'rm -f $HOME/docker/*.tar.gz'
-        - travis_wait docker-compose build test-ci
-        - >
-          mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}'
-          | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
     - stage: lint and jest
+      name: lint
       script:
-        - "travis_wait docker-compose run -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run lint && npm run jest'"
-    - stage: clean up cache
-      before_install: true
+        - npm run lint
+    - name: jest
```

Comment:
have you tested if it's faster to just run lint and jest in the same stage instead of clean installing twice?

Reply:
it's a minute lol

Reply:
oh good call.